wire-gradle-plugin/src/test/projects/lazy-configuration-resolution/gradle.properties

```diff
@@ -0,0 +1,2 @@
+org.gradle.configureondemand=true
+VERSION_NAME=5.4.0-SNAPSHOT
```

Comment:
this line might not be necessary?

Reply:
it's needed for https://github.com/square/wire/blob/cb4cfd74bd764de5f83fb06b374f65727920893d/wire-gradle-plugin/src/test/kotlin/com/squareup/wire/gradle/WirePluginTest.kt#L1431-L1432